internal/driver/driver.go

```diff
@@ -317,6 +317,12 @@ func (d *Driver) ExecuteReadCommands(device *Device, req sdkModels.CommandReques
 			return cv, errorWrapper.CommandError(command, err)
 		}
 		cv, err = sdkModels.NewCommandValue(req.DeviceResourceName, common.ValueTypeObject, data)
+	case VideoGetPixelFormat:
+		data, err = device.GetPixelFormat(cameraDevice)
+		if err != nil {
+			return cv, errorWrapper.CommandError(command, err)
```

Comment:
```suggestion
			return nil, errorWrapper.CommandError(command, err)
```

Reply:
But other commands in the same file are returning `cv` only which should be fine as it will be nil in case of error?

Reply:
In Marc's PR he is changing them all to `nil` so this will be consistent with that when merged